lib/the_little_thinkers_space/file_compressor.ex

```diff
@@ -49,7 +52,7 @@ defmodule TheLittleThinkersSpace.FileCompressor do
     |> Mogrify.save()
   end
 
-  defp compress_video(renamed_path, output_path) do
+  def compress_video(renamed_path, output_path) do
```

Comment:
I didn't see any changes that required this to be a public function, is this a mistake?

Reply:
yes, i needed it public to use it in the terminal